### ISSUE TITLE
Grid: Focus on clicked cell while editing row

### ISSUE THREAD

**Issue by prabh-62** [telerik/kendo-angular#2169]
**Describe the bug**
- While inline editing row in Kendo Grid, by default focus jumps to first cell.
	```typescript
	/**
     * Switches the specified table row in the edit mode ([see example]({% slug editing_template_forms_grid %}#toc-editing-records)).
     *
     * @param {number} index - The row index that will be switched in the edit mode.
     * @param {FormGroup} group - The [`FormGroup`]({{ site.data.urls.angular['formgroupapi'] }})
     * that describes the edit form.
     */
    public editRow(index: number, group: FormGroup): void {
        this.editService.editRow(index, group);
        this.focusEditElement(`tr[data-kendo-grid-item-index="${index}"]`);
    }
	```
- I used a *workaround* mentioned here to set the focus in the cell where user actually clicked: https://www.telerik.com/kendo-angular-ui-develop/components/grid/editing/editing-row-click/#toc-focusing-clicked-cells-while-editing

	```typescript
	//  Tests break if we run setTimeout within Angular Zone
	    this.zone.runOutsideAngular(() => {
	      setTimeout(() => {
	        this.zone.run(() => {
	          const inputElement: HTMLInputElement = document.querySelector(
	            `.k-grid-edit-row > td:nth-child(${columnIndex + 1}) input`
	          );
	          if (inputElement != null) {
	            inputElement.focus();
	          }
	        });
	      });
	    });
	```
- Now, when we have a kendo grid(10+ columns) with **horizontal scrollbars**, the view scrolls first to the extreme left(first cell) and then jumps to the clicked cell. This causes sudden jump in UI.

**To Reproduce**
Stackblitz: https://stackblitz.com/edit/angular-lfpshv

**Expected behavior**
Provide an argument for columnIndex
```typescript
  public editRow(rowIndex: number, columnIndex: number, group: FormGroup): void {

	}
```

**Browser**
 - OS: Windows
 - Browser Chrome
 - Version 73

**Comment by siyka-marinova** [comment]
Fix for this issue is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `kendo-angular-grid`

**Comment by Raisolution** [comment]
Available in `v4.7.0`

**Comment by dimitar-pechev** [comment]
To focus a specific cell on row editor open, provide a `columnIndex` value to the `editRow` method [as demonstrated here](https://www.telerik.com/kendo-angular-ui-develop/components/grid/editing/editing-row-click/#toc-managing-focus).
